blackbox/docs/administration/ssl.txt

```diff
@@ -0,0 +1,115 @@
+.. _administration_ssl:
+
+================================
+Secured Communications (SSL/TLS)
+================================
+
+.. NOTE::
+
+   *Secured Communications* is an Enterprise Edition feature.
+
+Secured communications allows you to encrypt traffic between CrateDB nodes
+and clients that use PostgreSQL Wire Protocol. Connections are secured using
+Transport Layer Security (TLS).
+
+SSL/TLS Configuration for PostgreSQL wire protocol
+==================================================
+
+First of all, you need to set the boolean setting ``ssl.psql.enabled`` to
+``true`` on each of the nodes you want to enable the secured communications.
+
+Configure the KeyStore
+----------------------
+
+SSL/TLS needs a keystore. The keystore holds the node certificate(s) which
+should be signed by a certificate authority (CA). A third-party CA or your
+organization's existing CA can be used.
+
+When a client connects to a node using SSL/TLS, the client receives the
+certificate provided by the node and will determine if the node’s certificate
+is valid, trusted, and matches the hostname or IP address it is trying to
+connect to.
+
+.. NOTE::
+
+    Technically, it's possible to disable CA checks for certificates on the
+    client. It's strongly recommended though to use certificates signed by an
+    official CA or by a private CA (company PKI) that is also known to the
+    client to be able to establish trust between nodes and clients.
```

Comment:
"client. This will help to ensure that establishing trust is as painless as possible." (wrapped)